packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -13,6 +13,10 @@ import useTabNav from './use-tab-nav';
 import useArrowNav from './use-arrow-nav';
 import useSelectAll from './use-select-all';
 import { store as blockEditorStore } from '../../store';
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
```

Comment:
Just nitpicking, but this should probably go to the top of the file, per coding guidlines.

Reply:
Yeah, fixed it in https://github.com/WordPress/gutenberg/pull/33446/commits/894848db647c888340bc6607b554b5e4cf8443cd.
The code autocomplete doesn't know where to put it 😄